core/templates/components/state-editor/state-editor.directive.ts

```diff
@@ -121,6 +121,28 @@ angular.module('oppia').directive('stateEditor', [
             $scope.conceptCardIsShown = !$scope.conceptCardIsShown;
           };
 
+          var questionSuggestionReview = angular.element(
+            document.querySelector('.oppia-question-suggestion-review'));
+          questionSuggestionReview.on('scroll', function() {
+            const stickyHeaderScrollTop = 50;
+            var tutorialStateContent = document.getElementById(
+              'tutorialStateContent');
+            var cardAvatarOnSticky = document.querySelector(
+              '.oppia-editor-card-avatar');
+            if (questionSuggestionReview.scrollTop() >
+              stickyHeaderScrollTop) {
+              tutorialStateContent.classList.remove(
+                'state-content-editor-parent-container');
+              cardAvatarOnSticky.classList.add(
+                'oppia-editor-card-avatar-sticky');
+            } else {
+              tutorialStateContent.classList.add(
+                'state-content-editor-parent-container');
+              cardAvatarOnSticky.classList.remove(
+                'oppia-editor-card-avatar-sticky');
+            }
+          });
```

Comment:
Please avoid the use of `angular.element` in the codebase -- it internally uses jQuery and we want to avoid that. We're not allowing this pattern going forward.